Automattic/jetpack
# Social: Add a 'more info' link to the editor upgrade nudge.

#### Changes proposed in this Pull Request:

The upgrade link in the editor drops the customer directly in the checkout screen. This PR adds a more info link that takes them to the main jetpack.com pricing page, with the Jetpack Social plan details open in a modal.

If they checkout from here they will be redirected back to the editor.

#### Other information:


#### Jetpack product discussion

1202583289493814-as-1203337382116439


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
~~This has highlighted an issue with the Jetpack redirects handling of query parameters. In order to test this properly you'll need to sandbox jetpack.com and apply D93475-code, until that or a similar fix is merged.~~

Although the Jetpack redirects issue still exists, we've changed the behaviour to work around it.

* Activate Jetpack Social on a test site, and ensure that the sharing limits are enabled and that there isn't a plan associated with the site.
* In the editor notice there is a 'More about...' link that opens a new tab with the pricing page on jetpack.com
* After purchasing a plan, you should be taken to the Jetpack Social admin page
* With Jetpack activated, the same process will take you to the default of the Jetpack recommendations admin page.
